src/components/20-molecules/file-upload/demo.js

```diff
@@ -90,6 +91,55 @@ storiesOf('Examples/File Upload/Pure HTML', module)
     `;
 
     const wrapper = document.createElement('div');
+    render(template, wrapper);
+    return wrapper;
+  })
+  .add('invalid', () => {
+    const wrapper = document.createElement('div');
+
+    setTimeout(() => {
```

Comment:
I'd prefer using litElement event listeners. You could just add `@click=${handleSubmit}` just like the `In a form` demo from the file upload does. This would be much cleaner.
https://lit-element.polymer-project.org/guide/events